### ISSUE TITLE
production grade charms need ntp

### ISSUE THREAD

**Issue by mbruzek** [mbruzek/layer-k8s#18]
Checking certificates require accurate time on the computer.  The NTP service must be installed on a production grade server.  It may already be on the cloud image but verify this and make sure it is installed, running, and properly configured.

**Comment by mbruzek** [comment]
This is more of a base layer issue and not related to kubernetes or certificates in general.  Possibly put this in the base layer or make a new "production" layer.

**Comment by mbruzek** [comment]
The stock cloud image does not appear to contain ntp.

`apt-get install -y ntp`

**Comment by lazypower** [comment]
or use the NTP subordinate

**Comment by mbruzek** [comment]
yes, we should do that, leave this comment here to remind us that any production bundle should include ntp
